examples/webgl_lines_fat.html

```diff
@@ -223,6 +226,59 @@
 
 			//
 
+			function onDocumentMouseMove( event ) {
+
+				if ( ! interactive ) return;
+
+				event.preventDefault();
+
+				var intersects = getIntersects( event.layerX, event.layerY );
+
+				if ( intersects.length > 0 ) {
+
+					if ( prevLinewidth === undefined ) {
+
+						prevLinewidth = matLine.linewidth
+
+					}
+
+					var res = intersects.filter( function ( res ) {
```

Comment:
Why not just using `intersects[ 0 ]`? Right now, you are creating in each call of `onDocumentMouseMove` an inline function.

Reply:
Sorry, redundant copypaste from webgl_raycast_sprite.html. Fixed in 96dc01126 .